HoI4Mod/Rome_Strikes_Back/common/national_focus/IND_NF.txt

```diff
@@ -211,8 +214,13 @@ focus_tree = {
 		ai_will_do = {
 			factor = 5
 		}
+		will_lead_to_war_with = BUR
 
 		completion_reward = {

```

Comment:
Generally better to provide the wargoal rather than a direct declaration, but either way is fine.